src/commands/pull.ts

```diff
@@ -21,21 +23,29 @@ async function fetchZipBlob(opts: PullOptions): Promise<Blob> {
     languages: opts.languages,
     filterState: opts.states,
     structureDelimiter: opts.delimiter,
+    filterNamespace: opts.namespaces,
   });
 }
 
 async function pullHandler(this: Command, path: string) {
   const opts: PullOptions = this.optsWithGlobals();
 
   await overwriteDir(path, opts.overwrite);
-
-  const zipBlob = await loading(
-    'Fetching strings from Tolgee...',
-    fetchZipBlob(opts)
-  );
-
-  await loading('Extracting strings...', unzipBuffer(zipBlob, path));
-  success('Done!');
+  try {
+    const zipBlob = await loading(
+      'Fetching strings from Tolgee...',
+      fetchZipBlob(opts)
+    );
+    await loading('Extracting strings...', unzipBuffer(zipBlob, path));
+    success('Done!');
+  } catch (e) {
+    if (e instanceof HttpError && e.response.status === 400) {
+      const res = await e.response.text();
+      error(res);
```

Comment:
This might be a little bit too brutal to dump the JSON error object to the user 😅

It'd be more friendly to show an appropriate error message in plain English explaining what's wrong.

Reply:
Thank you for your patient and detailed guidance, haha

Reply:
Super sorry for the late reply! Life is quite busy on my end 😅 

I think it'd be better to only return the error code instead of the raw json object, making it a tad more readable.

Reply:
@cyyynthia  Can I use this function "e.getErrorText()"
```
  if (e instanceof HttpError) {
      error(
        `Please check if your parameters, including namespaces, are configured correctly. Tolgee responded with error: ${e.getErrorText()}`
      );
      return;
    }
```

Reply:
This will not give the desired result; what you want is to get the body by using `await e.response.json()` and print the `obj.error` (I think those are the field names; I might be wrong but it's something along these lines. 🙂

Reply:
@cyyynthia  thanks for your suggestion, I have pushed a commit to optimize this code, this is the screenshot:
![Screenshot2023_10_16_103657](https://github.com/tolgee/tolgee-cli/assets/12186221/84d07e29-df65-4014-9860-95ccbb0903f2)

can you take a look when you are available

Reply:
This seems to be the expected error code, since the actual error reported by Tolgee is an empty export 👍🏻